packages/python-packages/api-stub-generator/apistub/nodes/_docstring_parser.py

```diff
@@ -6,6 +6,15 @@
 
 line_tag_regex = re.compile(r"^\s*:([^:]+):(.*)")
 
+default_patterns = [
+    # without quotes, cannot use period.
+    re.compile(r"\s+Default\s+value\s+is\s+([^\".]+)"),
```

Comment:
..."Defaults to azure.core.policies.DefaultRetryPolicy." would be busted, right? Would we support cross-linked typed references?

Reply:
(and, yes, the example is very much made up)

Reply:
Yes, it would be busted. You would have to use `Defaults to "azure.core.policies.DefaultRetryPolicy".`

Reply:
Though I suppose we could just do the capture with whitespace as the disallowed character and that would not necessitate quoting for dotted types.

Reply:
Well, that wasn't as smooth as I hoped. So for now you would use:
`defaults to "something.with.dots".`
Or:
`defaults to somethingwithoutdots.`

@johanste does that work?

Reply:
In other words, let's see how you'd write the doc comment in a way that renders well in sphinx and work backwards....

Reply:
So, despite sphinx endorsing this format, it doesn't actually make a difference in the sphinx-generated docs:
![image](https://user-images.githubusercontent.com/5723682/153672576-4e10eae1-bf25-413a-9ca4-644fd5c408d3.png)

So in that sense, this change does no harm.

Reply:
I changed the logic to no longer require quotations. 
https://apiviewstaging.azurewebsites.net/Assemblies/Review/00adead293274a5e8845d7468200c4bf

It will not parse sphinx annotations correctly, but does not (in this PR) make them clickable links. See issue: #2753